Automattic/Gravatar-SDK-iOS
# Prevent the session expired error state from changing on tap

Closes https://github.com/Automattic/Gravatar-SDK-iOS/issues/461

### Description

The session expired error state changes immediately after tapping on Login. It changes into Login required state. This looks a bit odd. Please try the testing steps on a different branch to check the problem.

It's happening because we remove the token immediately after tapping on Login. I am trying a different approach. Instead of removing the token i mark it as expired. Let's see how this works and whether if it's worth the extra logic or not.

### Testing Steps

- Disconnect your existing session https://wordpress.com/me/security/connected-applications
- Try opening QE with oauth flow
- You should see the session expired error state, tap login

Observe: The session expired message remains same
